### PR TITLE
N20: Release

### PR DESCRIPTION
**Pull request by ozkriff**
Preview (https://github.com/rust-gamedev/rust-gamedev.github.io/commit/4d55e74ad1e5a49680bc18037823f0a6493435b9): https://rust-gamedev-preview.netlify.app/posts/newsletter-020

Everyone is invited to review!

:timer_clock: Merging this in ~4/5 hours if there're no objections.

_Part of #531_

### CODE REVIEW

**Review by 17cupsofcoffee — APPROVED** [review]

File: content/posts/newsletter-020/index.md [hunk 1]
```diff
@@ -1060,14 +1036,14 @@ Version 0.5 is hot off the presses and rolls up features from the 0.4
...
+  archived and unarchived types.
 - Custom type bounds for serialization and deserialization can be added
-  with derive attributes
+  with derive attributes.
 - Helper types like [AlignedVec][rkyv-AlignedVec] and
-  [Infallible][rkyv-Infallible] were introduced to improve ergonomics
-- `const_generics` are now enabled by default
-- Helper functions have been added to make getting root objects easier
-- Several bugfixes and performance improvements
+  [Infallible][rkyv-Infallible] were introduced to improve ergonomics.
```
Comment (djkoloski) [inline content/posts/newsletter-020/index.md:1043]: were => was

**Review by djkoloski — CHANGES_REQUESTED** [review]
I used the wrong plurality, oops! Everything looks really good otherwise. 🙂

**Comment by kvark** [comment]
> e.g. posting issues on our github.

This contradicts the guidelines, I think. Shouldn't be written as "our".

> Station Iapetus  

Has both the image as a link, and the link in the image caption. Should the caption just say "click on the image to see the video", or the image be non-link instead?

> Emerald

The image here doesn't need to be so big, since it's just a logo, it doesn't show any actual rendering.

Honestly, this entry is also a bit fishy... The news for Emerald are just these two tiny lines:
>    rendering to textures
    WASM game sample.

While everything else is self-promotion and general information. I think the newsletter should put a stronger focus on the "news" part, and prevent parties from just using it to remind about themselves. I.e. something like "your news need to be at least as big as the general description" rule.

Similar concern could be applied to "Quinn" entry, although to a much lesser degree.

The "bevy" entry style is what I find to be the best, and every entry should try to follow the format.

In "wgpu" entry, something goes wrong about the image caption, it's not centered.

> (such as #[allow(unused_attributes)])

Would be nice to codify this part (i.e. with "`")

> based on the wgpu framework

nit: `wgpu` is a library, not a framework

File: content/posts/newsletter-020/index.md (hunk 1 above)
Comment (ozkriff) [reply content/posts/newsletter-020/index.md:1044]: :thinking: Are you sure? "Helper **types** like AlignedVec and Infallible **were** introduced to improve ergonomics"

Comment (djkoloski) [reply content/posts/newsletter-020/index.md:1044]: My bad, there's nothing to fix here. In my morning grog I missed the previous markdown line.

**Review by djkoloski — APPROVED** [review]

**Comment by ozkriff** [comment]
@kvark addressed most of the suggestions, thanks!

> The news for Emerald are just these two tiny lines:
> While everything else is self-promotion and general information. I think the newsletter should put a stronger focus on the "news" part, and prevent parties from just using it to remind about themselves. I.e. something like "your news need to be at least as big as the general description" rule.

This is the first newsletter issue that features Emerald's entry. It's ok that it focuses on a general overview.

> Similar concern could be applied to "Quinn" entry, although to a much lesser degree.

Same with Quinn.

**Comment by ozkriff** [comment]
:timer_clock:  Merging in 15 minutes.

**Comment by kvark** [comment]
Ok cool, I didn't realise this is the introduction for these libraries :)

**Pull request merged**